apps/console/src/features/roles/components/edit-role/edit-role-common/render-chip.tsx

```diff
@@ -32,23 +30,27 @@ interface RenderChipInterface extends IdentifiableComponentInterface, ChipProps
     /**
      * Callback to set the active option.
      */
-    setActiveOption: (option: GroupsInterface|UserBasicInterface) => void;
+    setActiveOption: (option: any) => void;
```

Comment:
Can we avoid using `any` here if possible?

Reply:
Here we are passing three different Interfaces `GroupsInterface`, `UserBasicInterface` and `ScopeInterface` hence will require to use `any`.

Reply:
Can we use | and add all the interfaces?